api/src/test/java/org/apache/iceberg/types/TestNestedFieldDefaultValues.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.types;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class TestNestedFieldDefaultValues {
+
+  @Test
+  public void testConstructors() {
+    int id = 1;
+    String fieldName = "fieldName";
+    Type fieldType = Types.IntegerType.get();
+    String doc = "field doc";
+    Integer defaultValue = 100;
+
+    // optional constructors
+    Assert.assertFalse(Types.NestedField.optional(id, fieldName, fieldType).hasDefaultValue());
```

Comment:
Nit: Statically importing `optional` and `required` will hugely improve readability.